cmd/orderimports/orderimports.go

```diff
@@ -0,0 +1,226 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// verify that all the imports have our preferred alias(es).
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/google/go-cmp/cmp"
+	"go/ast"
+	"go/build"
+	"go/parser"
+	"go/token"
+	"golang.org/x/term"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var (
+	regex      = flag.String("include-path", "cmd/kubeadm", "only files with paths matching this regex is touched")
```

Comment:
How about making it require the user to specify path to modify. We can make the tool work for all packages but initialy the verify scripts can target kubeadm only?